ldes-server-admin/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/admin/domain/validation/DcatShaclValidator.java

```diff
@@ -11,33 +12,28 @@
 
 @Component
 public class DcatShaclValidator extends AbstractShaclValidator {
-	private final String dcatShapeUri;
-
-	public DcatShaclValidator(@Value("${ldes-server.dcat-shape:}") String dcatShapeUri) {
-		this.dcatShapeUri = dcatShapeUri;
-	}
-
-	@Override
-	protected void initializeShapes() {
-		if (isValidatorActive()) {
-			shapes = Shapes.parse(Objects.requireNonNull(dcatShapeUri));
-		}
-	}
-
-	@Override
-	public boolean supports(Class<?> clazz) {
-		return Model.class.isAssignableFrom(clazz);
-	}
-
-	@Override
-	public void validate(Object target, Errors errors) {
-		if (isValidatorActive()) {
-			Model model = (Model) target;
-			validateShape(model);
-		}
-	}
-
-	private boolean isValidatorActive() {
-		return StringUtils.isNotBlank(dcatShapeUri);
-	}
+    private final String dcatShapeUri;
+
+    public DcatShaclValidator(@Value("${ldes-server.dcat-shape:}") String dcatShapeUri) {
+        this.dcatShapeUri = dcatShapeUri;
+    }
+
+    @Override
+    protected void initializeShapes() {
+        if (isValidatorActive()) {
+            shapes = Shapes.parse(Objects.requireNonNull(dcatShapeUri));
+        }
+    }
+
+    @Override
+    public void validate(@NotNull Object target, @NotNull Errors errors) {
```

Comment:
Do we still ever use this part of the interface? It looks like we only use it for unit testing?